app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt

```diff
@@ -125,6 +127,28 @@ class LoginPresenter(loginActivity: LoginActivity): ILoginPresenter, ILoginModel
         }
     }
 
+    override fun onSuccessSetting(response: Response<UserSetting>) {
+
+        loginView?.showProgress(false)
```

Comment:
in that case, user logs in before retrieving setting and app will not show retrieved setting until user close and open app again.